independent-projects/tools/devtools-common/src/test/java/io/quarkus/devtools/project/JavaVersionTest.java

```diff
@@ -31,7 +31,8 @@ public void givenJavaVersion21ShouldReturn21() {
 
     @Test
     void shouldProperlyUseMinJavaVersion() {
-        assertThat(getCompatibleLTSVersions(new JavaVersion("17"))).isEqualTo(JAVA_VERSIONS_LTS);
+        assertThat(getCompatibleLTSVersions(new JavaVersion("11"))).isEqualTo(JAVA_VERSIONS_LTS);
+        assertThat(getCompatibleLTSVersions(new JavaVersion("17"))).containsExactly(17, 21);
```

Comment:
@geoand that should answer your question, this is what is used to get the list of Java versions, the minimum is set in the Platform/Stream.

So no 11 won't be in the list if the minimum Java version of the stream is 17.